storybookjs/storybook
# Add "Quickstart" for Vue

# Issue
As a first-time user of Storybook, I had a difficult time figuring out how to get my knobs working with my Vue components. I eventually found what I needed to know by viewing the live-demo of the kitchen sink and seeing the code in the "Story" addon.

## What I did
Copied/Pasted the _With Angular_ section of the README and modified it to use the Vue-specific code.

## How to test
- Is this testable with Jest or Chromatic screenshots? **No.**
- Does this need a new example in the kitchen sink apps? **No.**
- Does this need an update to the documentation? **It _is_ an update to the documentation. 😄**

If your answer is yes to any of these, please make sure to include it in your PR.
